torch/csrc/jit/runtime/register_special_ops.cpp

```diff
@@ -216,6 +216,7 @@ RegisterOperators reg({
           return 0;
         },
         aliasAnalysisFromSchema()),
+    // not currently being generated, here for BC
```

Comment:
Maybe we should just have a `register_legacy_ops.cpp` so these aren't cluttering up the space